standard_reports/README.md

```diff
@@ -72,24 +72,26 @@ Optional fields in the manifest json are marked as '// optional'
 ### src dir
 
```

Comment:
Apologies for the noise in this file - I am struggling to match my linting for .md files.

Key changes are added diagram, and re wording of test and type generation set up.